src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3294,6 +3317,8 @@ bool SvgLoader::read()
 
     TaskScheduler::request(this);
 
+    if (!isBBoxGiven) this->done();
+
```

Comment:
@hermet @JSUYA 
this is an important change comparing to the #1248 
in case the isBBoxGiven == false, the vBox of the svg is known only after the whole file is handled.

Reply:
@mgrudzinska please remove the draft if you are ready to get a review.